specification/dimensions/tags.md

```diff
@@ -0,0 +1,76 @@
+# Tags
+
+The Tags column represents the set of tags assigned to Tag Sources that also account for potential provider-defined or user-defined tag evaluations.  Tags are commonly used for scenarios like adding business context to billing data to identify and accurately allocate charges.
+
+A tag becomes *finalized* when a single value is selected from a set of possible tag values assigned to the tag key.  This can occur when a tag is set at different levels of a resource grouping hierarchy (if supported by the Provider), or a tag is value is determined from a set of user-defined rules.
+
+The Tags column adheres to the following requirements:
+
+* The Tags column MUST contain user-defined and/or provider-defined tags.
+* The Tags column MUST only contain finalized tags.
+* The Tags column MUST be in Key-Value Format.
+* A Tag without a specified value MUST have its value set to null.
+* Providers MUST NOT alter user-defined Tag keys or values.
+
+Provider-defined Tags additionally adhere to the following requirements:
+
+* Provider-defined tags MUST be prefixed with a provider-specified tag key prefix.
+* Providers SHOULD publish all provider-specified tag key prefixes within their respective documentation.
+
+## Provider-Defined vs. User-Defined Tags
+
+The following is an example of one user-defined tag and one provider-defined tag, respectively, with tag key, `foo`.  The first tag, which is user-defined, is not prefixed. The second tag is prefixed with marketplace/ which the provider has specified as a reserved tag key prefix.
+
+```json
+    {
+        "foo":"bar",
+        "marketplace/foo": "bar"
+    }
+```
+
+## Finalized Tags
+
+A tag can either be static or dynamic. If a tag is static, its value is immutable. If a tag is dynamic, its value is determined by a set of predefined user or provider rules. A finalized tag is the final result of any static or dynamic Tag.
```

Comment:
Few minor modifications to @joshk0 's description. Ignore if they don't add value.
```suggestion
Within a provider, tag keys may be associated with multiple values, and potentially defined at different levels within the provider, such as accounts, folders, resources and other resource grouping constructs. When finalizing, providers must reduce these multiple levels of definition to a single value where each key is associated with exactly one value. The method by which this is done and the semantics are up to each provider, but must be documented within their respective documentation.
```